app.py

```diff
@@ -1,22 +1,58 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+import requests
+import os 
+from card import Card
 
 app = Flask(__name__)
-app.config.from_object('flask_config.Config')
 
 
 @app.route('/')
 def index():
-    items = session.get_items()
-    return render_template('index.html', items=items)
+    board_id = os.getenv('TRELLO_BOARD_ID')
+    board_key = os.getenv('TRELLO_KEY')
+    board_token = os.getenv('TRELLO_TOKEN')
+    items_response = requests.get(f'https://api.trello.com/1/boards/{board_id}/cards',params={'key': board_key,'token': board_token})
+    items_dictionary = items_response.json()
+    cards = []
+    for item in items_dictionary:
+        new_card = Card(item["id"], item["name"], item["idList"])
+        cards.append(new_card)
+
+    return render_template('index.html', items=cards)
 
 
 @app.route('/items', methods=['POST'])
 def add_item():
-    title = request.form['text-input']
-    session.add_item(title)
+    board_id = os.getenv('TRELLO_BOARD_ID')
+    board_key = os.getenv('TRELLO_KEY')
+    board_token = os.getenv('TRELLO_TOKEN')
```

Comment:
A bit repetitive getting these three variables again and again (sometimes unnecessarily, e.g. here `board_id` is unused). You could have a function that does it, like `get_trello_config`